packages/esm-patient-forms-app/src/constants.ts

```diff
@@ -6,3 +6,8 @@ export const customEncounterRepresentation = `custom:(uuid,encounterDatetime,enc
 
 export const formEncounterUrl = `${restBaseUrl}/form?v=custom:${customFormRepresentation}`;
 export const formEncounterUrlPoc = `${restBaseUrl}/form?v=custom:${customFormRepresentation}&q=poc`;
+
+export const formEngineResourceName = 'formEngine';
+export const htmlformentryFormEngine = 'htmlformentry';
+export const uiStyleResourceName = 'uiStyle';
+export const uiStyleSimple = 'simple';
```

Comment:
These are my first pass proposals for resource names and values that we would standardize on and program around.  Thoughts (@mogoodrich / @ibacher )?

Reply:
Open to other suggestions but works for me

Reply:
I think these all seem fine.